python/dgl/sampling/neighbor.py

```diff
@@ -52,6 +52,8 @@ def sample_neighbors(g, nodes, fanout, edge_dir='in', prob=None, replace=False,
         The features must be non-negative floats, and the sum of the features of
         inbound/outbound edges for every node must be positive (though they don't have
         to sum up to one).  Otherwise, the result will be undefined.
+    exclude_edges: tensor or dict
+        Edge IDs to exclude during sampling neighbors for the seed nodes.
```

Comment:
is it right that a tensor is for homogeneous graph and dict is for heterogeneous graph? if so, could you make it more explicit?